detox/src/servicelocator/android/index.js

```diff
@@ -21,5 +22,8 @@ AndroidServiceLocator.apkValidator = new ApkValidator(AndroidServiceLocator.aapt
 AndroidServiceLocator.fileXfer = new TempFileXfer(AndroidServiceLocator.adb);
```

Comment:
you might want to rename this one to tempFileXfer since we are using it as the real temp file xfer

Reply:
wth is `xfer`?? can't we use normal english? 😬

Reply:
Thinking about that "XFER" is a pretty common synonym for "File Transfer" 🙂 I guess we can call it `FileTransfer` or to leave it as is.. and to postpone this decision to another PR

Reply:
At least could you add JSDoc comment explaining the name... ?